lib/sync-records.ts

```diff
@@ -67,12 +67,34 @@ export const syncWorkbookRecords = async ({
     });
   }
 
+  type RetryData = [
```

Comment:
To confirm, RetryData is a tuple? And we expect `employeesWithoutManagersInDB` to be an arrow of these tuples?

Reply:
Yes a tuple. 

Edited type name to _UpsertEmployeeRetryData_